src/components/TasksTable/index.jsx

```diff
@@ -155,77 +202,120 @@ export default class TasksTable extends Component {
-        <DataTable
-          title="Bugs & Issues"
-          items={data}
-          renderRow={item => (
-            <TableRow tabIndex={-1} key={item.summary}>
-              <TableCell component="th" scope="row">
-                {item.project}
-              </TableCell>
-              <TableCell>
-                <List dense className={classes.summary}>
-                  <ListItem
-                    classes={{
-                      gutters: classes.summaryItem,
-                    }}
-                    button
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    component="a"
-                    href={item.url}>
-                    <ListItemText>{item.summary}</ListItemText>
-                    <LinkIcon size={iconSize} />
-                  </ListItem>
-                </List>
-              </TableCell>
-              <TableCell className={classes.tags}>
-                {item.tags.map(tag => (
-                  <Chip
-                    name={tag}
-                    key={tag}
-                    label={tag}
-                    className={classNames({
-                      [classes.clickedChip]: tag === query.tag,
-                    })}
-                    onClick={this.handleTagClick}
-                  />
-                ))}
-              </TableCell>
-              <TableCell>{item.assignee}</TableCell>
-              <TableCell>
-                {formatDistance(item.lastUpdated, new Date(), {
-                  addSuffix: true,
-                })}
-              </TableCell>
-            </TableRow>
-          )}
-          headers={['Project', 'Summary', 'Tags', 'Assignee', 'Last Updated']}
-          sortByHeader={sortBy}
-          sortDirection={sortDirection}
-          onHeaderClick={this.handleHeaderClick}
-          filters={
-            showFilterContent && (
-              <div className={classes.filter}>
-                <TextField
-                  select
-                  label="Assignee"
-                  value={assignment}
-                  className={classes.dropdown}
-                  onChange={this.handleFilterChange}>
-                  {assignments.map(assignee => (
-                    <MenuItem key={assignee} value={assignee}>
-                      {assignee}
-                    </MenuItem>
+      <Fragment>
+        <div className={classes.tableWrapper}>
+          <DataTable
+            title="Bugs & Issues"
+            items={data}
+            renderRow={item => (
+              <TableRow tabIndex={-1} key={item.summary}>
+                <TableCell component="th" scope="row">
+                  {item.project}
+                </TableCell>
+                <TableCell>
+                  <Button
+                    name={item.summary}
+                    className={classes.infoButton}
+                    onClick={this.handleDrawerOpen}>
+                    <InformationIcon />
+                  </Button>
```

Comment:
When the screen is resized to smaller dimensions the icon is no longer in the same row as the summary. Having the summary use '...' at the end when the summary is too long might be a possible fix. Also, I think you can specify the row to explicitly not allow wrapping with `white-space: 'nowrap'` on the cell.

Reply:
ok! i will do the `...` in other PR with the tooltips